firebase-firestore/src/main/java/com/google/firebase/firestore/local/RemoteDocumentCache.java

```diff
@@ -49,19 +47,18 @@
    * Looks up an entry in the cache.
    *
    * @param documentKey The key of the entry to look up.
-   * @return The cached Document or NoDocument entry, or null if we have nothing cached.
+   * @return The cached Document or NoDocument entry, or an invalid document if nothing is cached.
    */
-  @Nullable
-  MaybeDocument get(DocumentKey documentKey);
+  MutableDocument get(DocumentKey documentKey);
 
   /**
    * Looks up a set of entries in the cache.
    *
    * @param documentKeys The keys of the entries to look up.
-   * @return The cached Document or NoDocument entries indexed by key. If an entry is not cached,
-   *     the corresponding key will be mapped to a null value.
+   * @return The cached Document or NoDocument entries indexed by key. If an entry is not cached, an
+   *     invalid document is returned.
```

Comment:
This statement that an "invalid document is returned" is incorrect because the method returns a *Map*, not a *document*. The original comment can probably be tweaked, something like "If an entry is not cached, the corresponding key will be mapped to an invalid document."